source/includes/_uploading.md

```diff
@@ -51,18 +57,66 @@ func main() {
 ```
 
 Uploading a file to Skynet can be done through a Skynet portal or your
-local siad instance.
+local `siad` instance.
 
 <aside class="notice">
 If a file is uploaded through a portal, the portal owner is paying to host that file, and it will remain on the network for as long as that file contract is valid.
 </aside>
 
-### Settings
+### Parameters
+
+Field | Description
+----- | -----------
+`path` | The local path where the file to upload may be found.
```

Comment:
that is not going to be true for browser sdk as browsers need to use `File` object instances